src/job-execution/WebJobExecutor.ts

```diff
@@ -101,20 +101,9 @@ export class WebJobExecutor extends BaseJobExecutor {
             this.appendRequest(res, sasJob, config.debug)
             resolve(jsonResponse)
           }
-          if (this.serverType === ServerType.Sas9 && config.debug) {
-            const jsonResponse = parseWeboutResponse(res.result as string)
```

Comment:
Why do we not need to do this anymore? Does the server not respond with a string containing `>>weboutBEGIN<<` and `>>weboutEND<<`?

Reply:
httpClient is doing the parse 
![image](https://user-images.githubusercontent.com/18329105/126874637-c2dcfb16-05f4-4d46-8fa6-38d8d8ede808.png)
![image](https://user-images.githubusercontent.com/18329105/126874652-fe6784ef-0a05-4f79-b863-7a4d21910697.png)